metadata-ingestion/src/datahub/emitter/rest_emitter.py

```diff
@@ -228,11 +233,15 @@ def emit_mce(self, mce: MetadataChangeEvent) -> None:
         snapshot_fqn = (
             f"com.linkedin.metadata.snapshot.{mce.proposedSnapshot.RECORD_SCHEMA.name}"
         )
+        ensure_mce_has_system_metadata(mce)
         system_metadata_obj = {}
         if mce.systemMetadata is not None:
             system_metadata_obj = {
```

Comment:
why can't we just call mce.systemMetadata.to_obj() here? I don't want us to be maintaining this trivial mapping

Reply:
Makes sense. Done.